README.md

```diff
@@ -49,9 +49,10 @@ List of content:
 * [Staketab pool](https://staketab.com/) A staking pool with 7% fee
 * [ZKValidator pool](https://zkvalidator.com/) A staking pool with 10% fee
 * [P2P Validator](https://p2p.org/) A staking pool with 8% fee
+* [TheMinaPool](https://theminapool.com) A staking pool with 5% fee and open source payout
```

Comment:
Since staking pools might end up turning controversial, and in order to prevent a spammy promotional board, I would suggest to everybody stick to the same format `name + fees` 🙏

Reply:
sorry for being spammy :) will revert.